Yoast/wpseo-woocommerce
# Make sure we analyze the excerpt content too

## Summary

This PR can be summarized in the following changelog entry:

* Analyze the content of the short description as part of the page content.

## Test instructions

This PR can be tested by following these steps:

* Edit the short description of a product, see your words are counted in the analysis.

Fixes #73
